app/services/mapping/declaration/GoodsItemQuantityBuilder.scala

```diff
@@ -41,7 +40,7 @@ object GoodsItemQuantityBuilder {
 
   def build(implicit cacheMap: CacheMap): DeclarationGoodsItemQuantityType =
     cacheMap
-      .getEntry[Seq[GovernmentAgencyGoodsItem]](ExportsItemsCacheIds.itemsId)
+      .getEntry[Seq[GovernmentAgencyGoodsItem]]("exportItems")
```

Comment:
Why this change? I think it, it's better to have value in val, to change this if necessary only in one place

Reply:
ExportsItemsCacheIds was a class inside itemscachingService. In reality this reference is going to be deleted as the cachemap / exportItems reference will cease to exist once the old caching is full removed. Decided to do this as a quick fix so the code compiles.